src/Examine.RemoveDirectory/RemoteIndexOutputFactory.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Lucene.Net.Store;
+
+namespace Examine.RemoveDirectory
```

Comment:
RemoveDirectory?

Reply:
@nzdev I am guessing it was already that few beers, which I drunk that evening, going to fix that asap :)

Reply:
@nzdev name was correct, any other comments :)